backend/src/LeanCode.AppRating/CQRS/SubmitAppRatingCH.cs

```diff
@@ -62,6 +71,8 @@ public Task ExecuteAsync(HttpContext context, SubmitAppRating command)
                 )
             );
 
+        publishEndpoint.Publish(new RateSubmitted<TUserId>(userId, command.Rating));
```

Comment:
Open question, but could we save a bit of complexity by converting UserId to string before publishing this message to avoid generics in its type? Or do you expect consumers of this package to provide their own, well, consumers, which might prefer the TUserId type to stay intact?

Reply:
I believe leaving it generic here is open for new consumers in the future that may need the generic type of the UserId